src/main/resources/InstructorSampleData.json

```diff
@@ -686,20 +686,23 @@
       ]
     },
     "qn1InSession3": {
-      "feedbackSessionName": "Second team feedback session",
+      "feedbackSessionName": "Second team feedback session (point-based)",
       "courseId": "demo.course",
       "questionDetails": {
-        "questionText": "How much work did each team member contribute? (response will be shown anonymously to each team member).",
-        "questionType": "CONTRIB"
+        "distributeToRecipients": true,
+        "pointsPerOption": true,
+        "questionText": "Distribute points among team members based on their contributions on the user documentation so far.",
+        "questionType": "CONSTSUM",
+        "points": 100,
+        "constSumOptions": []
       },
+      "questionDescription": "If a team member did an equal share of the work, give 100 points. If a team member did about 10% more than an equal share of the work, give 110 points, and so on",
```

Comment:
Description should end with a period (or maybe an ellipsis). Apologies for bringing this up so many times, a bit more particular about the minor details this time because any typos/grammatical mistakes in the sample data gives a bad impression.

Another point, I noticed this is the only question with a filled description. Is this intentional?

Reply:
Yup this is intentional, this is an added description because the question type that fits such a question best is the contribution question, since we are using another question type(constant sum) which is not made specific to such aa scenario the original issue included this description

Reply:
In that case, do you think there's value in adding this description to the template as well?

Reply:
Amended!